### PR TITLE
Update services form

### PR DESCRIPTION
**Pull request by blackinno**
- Update webhook service form, Closes https://github.com/mesg-foundation/aragon/issues/17.
- Update new connection form, Closes https://github.com/mesg-foundation/aragon/issues/18
- Update slack service form, Closes https://github.com/mesg-foundation/aragon/issues/19
- Update telegram service form, Closes https://github.com/mesg-foundation/aragon/issues/21

### CODE REVIEW

**Comment by NicolasMahe** [comment]
Seems to be fine from looking at the code.
Is it possible to see this online (if it doesn't take too much time)? Or at least see some screenshot ;)

**Comment by blackinno** [comment]
> Seems fine from looking at the code.
> Is it possible to see this online (if it doesn't take too much time)? Or at least see some screenshot ;)

You can look at this [link](https://rinkeby.aragon.org/#/0x702E819000FC3Be0dadC99480401bD94DBD5b322/0xf390905ab56bf9ed27fbc9d24ead1b46c8053ec5/).

File: app/src/components/AppConnection/ServiceDataForm/ServiceForm.js [hunk 1]
```diff
@@ -54,13 +54,34 @@ class ServiceForm extends Component {
...
+          <div
+            css={`
+              margin-top: 5px;
+            `}
+          >
+            <Info>
+              {service?.description?.text}{' '}
+              {service?.description?.link ? (
+                <a href={service?.description?.link} target='_blank' rel='noopener noreferrer'>
+                  example
```
Comment (NicolasMahe) [inline app/src/components/AppConnection/ServiceDataForm/ServiceForm.js:68]:
```suggestion
                  (example)
```
---
Note: It's not nice to hardcode the text of the link here

Comment (blackinno) [reply app/src/components/AppConnection/ServiceDataForm/ServiceForm.js:68]: Fix on https://github.com/mesg-foundation/aragon/pull/25

**Pull request merged**